docs/tutorials/variables/regular.rst

```diff
@@ -289,9 +292,10 @@ Name of variable may consist of **any** characters:
 Quotes
 ======
 
-In previous example quote was used to create name with space - this is called
-*quoted argument*. Note that argument should start and end with quote, otherwise
-it became *unquoted argument* so quote will be treated as part of the string:
+In the previous example, the quote character `"` was used to create a name containing 
+a space - this is called *quoted argument*. Note that the argument must start and end
+with a quote character, otherwise it becames an *unquoted argument*. In this case, the
```

Comment:
Spellchecker [complains](https://travis-ci.org/ruslo/CGold/builds/161078722#L715) about `becames` word. I guess since it's `become`/`became`/`become` hence there is no `becames`, it should be `becomes` or `became` (?)